Python/initconfig.c

```diff
@@ -280,39 +254,27 @@ static const char usage_envvars[] =
 "PYTHON_HISTORY  : the location of a .python_history file.\n"
 "\n"
 "These variables have equivalent command-line options (see --help for details):\n"
-"PYTHON_CPU_COUNT: Overrides the return value of os.process_cpu_count(),\n"
-"                  os.cpu_count(), and multiprocessing.cpu_count() if set to\n"
-"                  a positive integer. (-X cpu_count)\n"
+"PYTHON_CPU_COUNT: override the return value of os.cpu_count() (-X cpu_count)\n"
 "PYTHONDEBUG     : enable parser debug mode (-d)\n"
-"PYTHONDEVMODE   : enable the development mode (-X dev)\n"
+"PYTHONDEVMODE   : enable the Python development mode (-X dev)\n"
```

Comment:
Capitalised at https://docs.python.org/3/library/devmode.html, can skip "the"
```suggestion
"PYTHONDEVMODE   : enable Python Development Mode (-X dev)\n"
```